src/main/kotlin/com/stake/mercariapplication/service/LoginService.kt

```
@@ -0,0 +1,26 @@
+package com.stake.mercariapplication.service
+
+import com.stake.mercariapplication.repository.UserRepository
+import com.stake.mercariapplication.util.Logger
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.security.crypto.password.PasswordEncoder
+import org.springframework.stereotype.Service
+
+@Service
+class LoginService() {
```

detekt

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)